vpr/src/route/route_util.cpp

```diff
@@ -27,6 +29,13 @@ vtr::Matrix<float> calculate_routing_usage(t_rr_type rr_type, bool is_flat) {
 
     //Record number of used resources in each x/y channel
     for (RRNodeId rr_node : rr_nodes) {
+        if (!is_print) {
+            t_draw_state* draw_state = get_draw_state_vars();
```

Comment:
Wrap in a 
#ifndef NO_GRAPHICS

#endif

for this whole if statement.  Otherwise it won't compile when we're trying to compile with NO_GRAPHICS as all the drawing functions and gtk interface have been stripped out.